### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by Blahartinger**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Blahartinger/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+showing by default. Feel free to look around.
+
+Over on the right is a panel for line notes that you can connect to a
+line of code in the diff. You can also switch to the conversation
+to see comments on the whole pull request.
+
+Line notes have flags to indicate what is going on in the code,
+too. Flags are as follows:
+
+`['!fix', '!resolve']`
```
Comment (Blahartinger) [inline ReadMe.md:15]: cool

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+- Something in the code needs to be reviewed and fixed before
+  the pull request is merged.
+
+`['!fixed', '!resolved', '!completed']`
+- The issue has been resolved.
+
+To add a flag simply leave a comment with your desired flag. For
```
Comment (Blahartinger) [inline ReadMe.md:22]: needs to be fixed !fix

File: ReadMe.md [hunk 3]
```diff
@@ -1 +1,41 @@
...
+convenience we also have a dropdown menu to add these comments
+automatically.
+
+Another key feature you need to know about is ninja stars. Ninja 
+stars are how you communicate that a pull request is okay to merge.
```
Comment (Blahartinger) [inline ReadMe.md:27]: !fixed now its fixed

**Pull request merged**